tensorflow/tensorboard/components/tf-event-dashboard/tf-event-dashboard.html

```diff
@@ -110,6 +110,12 @@
                       icon="fullscreen"
                       on-tap="toggleSelected"
                     ></paper-icon-button>
+                      <paper-icon-button
```

Comment:
To make this slightly more self-documenting, please add a title text like "Toggle between Linear and Logarithmic y scale"

Or you can make a title like "swap to log scale" and then change the title when the button is clicked.